src/main/resources/app.config

```diff
@@ -0,0 +1,5 @@
+app.jdbcMySQLURL=jdbc:mysql://localhost:3306/
```

Comment:
You need to move your database to the remote MYSQL server

Reply:
Done